Use average velocity field for multiphysics (#534)
lethe-cfd/lethe: doc/source/parameters/cfd/multiphysics.rst

    set heat transfer 		= false
    set viscous dissipation 	= false
    set buoyancy force 		= false

>     set use average velocity field =false

Reviewer: ```suggestion
    set use average velocity field = false
```

* ``VOF``: enables multiphase flow simulations, with two fluids separated by a free surface, using the Volume-of-Fluid method. 

  See :doc:`volume_of_fluid` for advanced VOF parameters, :doc:`initial_conditions` for the definition of the VOF conditions and `Physical properties - two phase simulations <https://lethe-cfd.github.io/lethe/parameters/cfd/physical_properties.html#two-phase-simulations>`_ for the definition of the physical properties of both fluids.
  

> * ``use average velocity field``: controls if the advection of the physics is done using the average velocity field instead of the current velocity field.

Reviewer: ```suggestion
* ``use average velocity field``: controls if the advection of the subphysics is done using the average velocity field instead of the current velocity field.
```

Reviewer: Add a comment stating when it can be used? (This is useful when...)

.. note::

> 	At this point, the average velocity field is only implemented in the heat transfer physic.

Reviewer: ```suggestion
	Currently, this mechanism is only implemented in the heat transfer physic.
```

> 

Reviewer: ```suggestion

.. important::
   The subphysic must also be enabled (for example, ``set heat transfer = true``)

```

* ``VOF``: enables multiphase flow simulations, with two fluids separated by a free surface, using the Volume-of-Fluid method. 

  See :doc:`volume_of_fluid` for advanced VOF parameters, :doc:`initial_conditions` for the definition of the VOF conditions and `Physical properties - two phase simulations <https://lethe-cfd.github.io/lethe/parameters/cfd/physical_properties.html#two-phase-simulations>`_ for the definition of the physical properties of both fluids.
  

> * ``use average velocity field``: controls if the advection of the subphysics is done using the average velocity field instead of the current velocity field. This is useful when the flow dynamics and the subphysics reach a pseudo-steady state at different time scales.

Reviewer 2: Also, I think the parameter could have a more meaningful name. My suggestions would be "use time averaged velocity field", "use steady state velocity field" or "use constant velocity field". The "average" gives an idea of spatial average instead of temporal average.

> * ``use average velocity field``: controls if the advection of the subphysics is done using the average velocity field instead of the current velocity field. This is useful when the flow dynamics and the subphysics reach a pseudo-steady state at different time scales.

Reviewer 3: Agreed. Use time-averaged would be the best way to write it.
